great_expectations/checkpoint/checkpoint.py

```diff
@@ -76,6 +76,25 @@ def __init__(
             raise TypeError("A Checkpoint requires a valid DataContext")
         self._data_context = data_context
 
+        # if checkpoint was created without yml
+        if validations:
+            for idx, val in enumerate(validations):
```

Comment:
I like the use of enumerate to handle indexing when there are multiple validations present. Would it make sense to have more than 1 validation in the test so that we can ensure no regressions in this part of the code?

Reply:
I :heart: this idea. I added a new test to be clear about what is being tested.